packages/core/test/unit/initializer/StrykerInitializer.spec.ts

```diff
@@ -52,6 +53,7 @@ describe(StrykerInitializer.name, () => {
       .provideClass(initializerTokens.npmClient, NpmClient)
       .provideValue(initializerTokens.strykerPresets, presets)
       .provideClass(initializerTokens.configWriter, StrykerConfigWriter)
+      .provideClass(initializerTokens.gitignoreWriter, GitignoreWriter)
```

Comment:
I'm pretty sure this will result in the real one being included. Please use a mocked `GitingoreWriter` and test that it is called correctly.